packages/components/src/Payments/PaymentsEth.tsx

```diff
@@ -0,0 +1,177 @@
+import * as React from "react";
+import { Mutation, MutationFunc } from "react-apollo";
+import { EthAddress } from "@joincivil/core";
+import { PAYMENTS_ETH_MUTATION } from "./queries";
+import { UsdEthConverter } from "../";
+import {
+  PaymentDirectionsStyled,
+  PaymentEthLearnMore,
+  PaymentBtn,
+  PaymentAmountEth,
+  PaymentTypeLabel,
+} from "./PaymentsStyledComponents";
+import {
+  PayWithEthText,
+  WhyEthInfoText,
+  WhatIsEthInfoText,
+  CanUseCVLInfoText,
+  ConnectWalletWarningText,
+  ConnectMobileWalletModalText,
+  PayWithEthDescriptionText,
+} from "./PaymentsTextComponents";
+import { PaymentsModal } from "./PaymentsModal";
+import { PaymentsEthForm } from "./PaymentsEthForm";
+
+export enum MODEL_CONTENT {
+  WHY_ETH = "why eth",
+  WHAT_IS_ETH = "what is eth",
+  CAN_USE_CVL = "can I use cvl",
+}
+
+export interface PaymentsEthProps {
+  postId: string;
+  newsroomName: string;
+  paymentAddress: EthAddress;
+  userAddress?: EthAddress;
+  usdToSpend: number;
+  isWalletConnected: boolean;
+  handlePaymentSuccess(): void;
+}
+
+export interface PaymentsEthStates {
+  isMobileWalletModalOpen: boolean;
+  isInfoModalOpen: boolean;
+  modalContent: string;
```

Comment:
Minor, but this type could be `MODEL_CONTENT` (I guess `modalContent?: MODEL_CONTENT`) so that you can set it to `undefined` for no modal showing.